flybywiresim/aircraft
# fix: nd bugs and paths

Fixes #8445 
Fixes #8449 
Fixes #8438 

## Summary of Changes

Some further regressions and bugs in NDv2 that have been reported.

Namely:
ADF needles never showing after a C&D start
ADF and VOR needles not having a clip mask
Selected heading bug being shown in managed heading mode when switching to TRK/FPA and vice versa

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. ADF needles never showing after a C&D start
  1. Spawn C&D, tune a valid NDB (e.g. KEK or 403.0). Make sure the ADF needles appear on the ND when the ADIRS are not aligned and also after alignment). 
 1. Ensure the VOR and ADF needles never overlay the chrono/radio information on the ND (mainly lower left corner)

1. Selected heading bug being shown in managed heading mode when switching to TRK/FPA and vice versa
 1. Try the reproduction steps from https://github.com/flybywiresim/aircraft/issues/8449 and make sure the selected heading bug does not show (you must be airborne).


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
